doc/opt-modes.md

```diff
@@ -0,0 +1,63 @@
+# kubemacpool Opt-Modes
+
+Kubemacpool supports Opt-In and Opt-Out opt-modes, in order to control whether MAC Address will 
+be allocated to all VirtualMachines and Pods by default or not.
+The opt-mode can be set for Pods and VirtualMachines separately.
```

Comment:
I believe it would be useful to add a paragraph explaining the behaviour of opt-in and opt-out, what do you think?

Reply:
Added. Done